runtime/src/bank.rs

```diff
@@ -5259,7 +5260,10 @@ impl Bank {
         self.loaded_programs_cache
             .write()
             .unwrap()
-            .sort_and_unload(Percentage::from(SHRINK_LOADED_PROGRAMS_TO_PERCENTAGE));
```

Comment:
sort_and_unload is now dead code and can probably be removed right?

Reply:
Left it in as an escape hatch. We can remove it once the new eviction policy is live and working on MB.